opm/autodiff/FlowMainEbos.hpp

```diff
@@ -282,24 +282,32 @@ namespace Opm
             logFileStream << ".PRT";
             debugFileStream << ".DEBUG";
 
-            std::string debugFile = debugFileStream.str();
```

Comment:
A few comments here:
 - I think that the STREAMLOG also should get a separate scope, and perhaps an option to disable, too?
 - There is a chicken-and-egg problem with the parameters writing to the (debug) log, while the logs depend on parameters. This makes it a little worse (won't see the no_*_log options in .DEBUG) but not so much that it disqualifies the PR.
 - The logFile_ member is only used to later give a message on line 405 (current master numbering), perhaps it should be removed, or perhaps set to empty, so that that message can avoid giving untruthful information (look at the nonexisting log).